src/components/Search.jsx

```diff
@@ -219,15 +211,15 @@ export const Search = ({
                     <li className="search-footer pointer-events sticky bottom-0 flex space-x-5 p-2 text-sm text-gray-dark">
                         <span className="flex items-center">
                             <Key char="up" className="mr-0.5" />
-                            <Key char="down" className="mr-2" /> to navigate
+                            <Key char="down" className="mr-2" /> navigate results
                         </span>
 
                         <span className="flex items-center">
-                            <Key char="enter" className="mr-2" /> to submit
+                            <Key char="enter" className="mr-2" /> purchase details
```

Comment:
this is a common search across all apps, i think its better to not use `purchase` in this, as this may be used in other places.

Reply:
@anush This is a copy change you had suggested, reverting this based on Manoj's comment. FYI